mobile/src/package.json

```diff
@@ -30,8 +31,23 @@
   },
   "devDependencies": {
     "@babel/core": "^7.8.6",
+    "@babel/plugin-transform-flow-strip-types": "^7.10.4",
+    "axios-mock-adapter": "^1.18.2",
     "babel-preset-expo": "^8.2.3",
-    "prettier": "^2.1.2"
```

Comment:
why did we get rid of prettier?

Reply:
oh nvm u just moved it lol

Reply:
@trentstauff lmao I didn't know I added you as a review